apps/src/templates/MultipleAssignButton.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import Button from './Button';
+import i18n from '@cdo/locale';
+import {
+  assignToSection,
+  unassignSection,
+  sectionsForDropdown
+} from '@cdo/apps/templates/teacherDashboard/teacherSectionsRedux';
+import MultipleSectionsAssigner from '@cdo/apps/templates/MultipleSectionsAssigner';
+import {sectionForDropdownShape} from '@cdo/apps/templates/teacherDashboard/shapes';
+
+class MultipleAssignButton extends React.Component {
+  static propTypes = {
+    sectionId: PropTypes.number.isRequired,
+    sectionName: PropTypes.string,
+    courseId: PropTypes.number,
+    courseOfferingId: PropTypes.number,
+    courseVersionId: PropTypes.number,
+    scriptId: PropTypes.number,
+    assignmentName: PropTypes.string,
+    reassignConfirm: PropTypes.func,
+    isOnCoursePage: PropTypes.bool,
+    isStandAloneUnit: PropTypes.bool,
+    participantAudience: PropTypes.string,
+    // Redux
+    assignToSection: PropTypes.func.isRequired,
+    isRtl: PropTypes.bool,
+    sectionsForDropdown: PropTypes.arrayOf(sectionForDropdownShape).isRequired
+  };
+
+  state = {
+    assignmentChoiceDialogOpen: false
+  };
+
+  onCloseDialog = () => {
+    this.setState({
+      assignmentChoiceDialogOpen: false
+    });
+  };
+
+  handleClick = () => {
+    this.setState({
+      assignmentChoiceDialogOpen: true
+    });
+  };
+
+  render() {
+    const {assignmentChoiceDialogOpen} = this.state;
+    const {
+      courseId,
+      courseOfferingId,
+      courseVersionId,
+      scriptId,
+      assignmentName,
+      isStandAloneUnit,
+      isRtl,
+      sectionsForDropdown,
+      participantAudience,
+      isOnCoursePage
+    } = this.props;
+
+    // Adjust styles if locale is RTL
+    const buttonMarginStyle = isRtl
+      ? styles.buttonMarginRTL
+      : styles.buttonMargin;
+
+    return (
+      <div>
+        <div style={buttonMarginStyle}>
+          <Button
+            __useDeprecatedTag
```

Comment:
@hannahbergam -- do you know what the current guidance is for using the `__useDeprecatedTag` prop is for `Button`?

Reply:
Hi! Thanks for tagging me! Adding this tag makes the button non-accessible via tab navigation. We are only currently okay with using this for links (href=), but are removing the tag from all 'pure button' instances (no href, only onClick=, which is the case here). Does this make sense? Removing the line should only change the box shadowing of the button, which can be added back in if you are trying to match styling with another nearby button. Otherwise, we are moving toward 'flat' buttons anyway (no box shadow). Happy to give more info if I'm being unclear!

Reply:
That does make sense - thanks for the explanation!